README.md

```diff
@@ -9,7 +9,7 @@ This chart bootstraps WebSight CMS deployment on a Kubernetes cluster using the
 
 - Kubernetes cluster with at least 8 GB ram and 2 CPU
 - Ingress installed on k8s cluster (you may use e.g. [Nginx ingress](https://kubernetes.github.io/ingress-nginx/deploy/)).
-- Helm 3.6+
+- Helm 3.8+
```

Comment:
OCI standard is officially supported since Helm 3.8: https://helm.sh/docs/topics/registries/